Automattic/jetpack
# Jetpack Setup Wizard: Add Setup Wizard status and routing memory

#### Changes proposed in this Pull Request:

This PR improves upon the Jetpack Setup Wizard by adding the functionality to remember which step the user was on if they leave and return to the Setup Wizard, and by hiding the Setup Wizard after "Complete" has been pressed.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
Project: p1HpG7-8BF-p2
Design: pbtFPC-ol-p2

#### Does this pull request change what data or activity we track or use?



#### Testing instructions:





1. Add the following to the end of wp-config.php: add_filter( 'jetpack_show_setup_wizard', '__return_true' );
2. Visit `/wp-admin/admin.php?page=jetpack#/setup` and verify that you are routed to `#/setup/intro`.
3. Navigate to the dashboard and return to the wizard. Verify that you are still routed to `#/setup/intro`.
4. Advance through the questionnaire and repeat for the other pages.
5. Click "I'm done for now" and verify that you are taken to the Dashboard.

#### Proposed changelog entry for your changes:

* None, there should be one entry for the entire Setup Wizard.
